tests/conftest.py

```diff
@@ -102,7 +103,7 @@ def test_ep() -> None:
 [project]
 name = "testproject"
 description = "an example test project for testing vulcan builds, %"
-authors = [{name="Joel Christiansen", email="joelchristiansen@optiver.com"}]
+authors = [{{name="Joel Christiansen", email="joelchristiansen@optiver.com"}}]
```

Comment:
why?

Reply:
This is now being `.format`ed, so the literal curly braces need to be escaped, which is done by doubling up